src/components/Records/Record/GeneralInfo/SavedSearches.vue

```diff
@@ -0,0 +1,169 @@
+<template>
+  <div
+    v-if="getField('savedSearches') && getField('savedSearches').length"
+  >
+    <!--      <v-chip-->
+    <!--        v-for="search in getSavedSearches"-->
+    <!--        :key="search.id"-->
+    <!--        class="ma-1"-->
+    <!--        color="secondary"-->
+    <!--        :close="user().is_super_curator ? true : false"-->
+    <!--        close-icon="mdi-delete"-->
+    <!--        text-color="white"-->
+    <!--        @click:close="confirmDeleteSavedSearch(search)"-->
+    <!--      >-->
+    <!--        <a-->
+    <!--          class="white&#45;&#45;text"-->
+    <!--          :href="search.url"-->
+    <!--        >-->
+    <!--          {{ search.name }}-->
+    <!--        </a>-->
+    <!--      </v-chip>-->
```

Comment:
Is this block something which might be needed in future?

Reply:
Not needed anymore, hence removed.